react/components/searchbar.js

```diff
@@ -117,8 +117,12 @@ const SearchBar = React.createClass({
     this.props.setQuery(query);
   },
   handleUserChange() {
-    var value = this.refs.user.value;
-    var query = { ...this.props.query, user: value };
+    // lowercase all user names
+    // since this seems to be how we index them
+    // in elasticsearch
+    const value = `${this.refs.user.value}`.toLowerCase()
+    const rawValue = this.refs.user.value
+    const query = { ...this.props.query, user: value, userRaw: rawValue }
```

Comment:
I couldn't quickly tell if this username search bug was present before with Elasticsearch 2.3.4, so I went ahead and fixed it here 😄

Reply:
standalone PR for this here, for good measure https://github.com/enjalot/blockbuilder-search/pull/41